symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/util/StreamUtil.java

```diff
@@ -0,0 +1,45 @@
+package com.symphony.bdk.core.service.stream.util;
+
+import org.apiguardian.api.API;
+
+import java.util.Base64;
+
+/**
+ * Helper class providing stream id conversion
+ */
+@API(status = API.Status.EXPERIMENTAL)
+public class StreamUtil {
+
+  private StreamUtil() {
+  }
+
+  /**
+   * Convert the stream id to the corresponding URLSafe encoded stream id
+   *
+   * @param streamId of the stream to be parsed
+   * @return stream id after conversion
+   */
+  public static String toUrlSafeStreamId(String streamId) {
+    if (streamId == null || streamId.isEmpty()) {
+      return "";
+    }
+    return streamId.trim()
```

Comment:
don't we want to use the same logic as https://github.com/SymphonyOSF/SymphonyLibrary/blob/master/foundation/src/main/java/com/symphony/streams/StreamId.java#L58
(to avoid regex or at least rely on some existing library)

Reply:
you are right! Changed to use the same logic in there.